include/rocksdb/db.h

```diff
@@ -1421,6 +1421,9 @@ class DB {
 
   // Increase the full_history_ts of column family. The new ts_low value should
   // be newer than current full_history_ts value.
+  // If another thread updates full_history_ts_low concurrently to a higher
+  // timestamp than the requested ts_low, a try again error will be returned.
+  // Data between [ts_low, full_history_ts_low) is not kept.
```

Comment:
Note that `IncreaseFullHistoryTsLow()` does not actually remove data, so I think we can remove the last sentence.

Reply:
Cool, removed this line.